README.md

```diff
@@ -15,4 +15,14 @@
 ### `npm run build`
 
 Собирает silaederdashboard в `build` папку.\
-Эта команда автоматически оптимизирует React для деплоинга silaederdashboard.
\ No newline at end of file
+Эта команда автоматически оптимизирует React для деплоинга silaederdashboard.
+
+## Для разработчиков:
+
+### Установка линтера:
+
+```
+git add .husky/pre-commit
+```
+
+Теперь линтер будет проверять код перед каждым коммитом.
```

Comment:
Btw, I don't think this actually has any effect on running linter

Reply:
It's for running linter before you commit changes. See [Husky](https://github.com/typicode/husky) for more info

Reply:
Well, I cloned your repo, run this line and then created a commit. Nothing happened, while I would expect an error, as I don't even have npm/npx installed

Reply:
I've updated readme, so it now says, that you have to have npm and nodejs installed on your system